microsymposium.dic

```diff
@@ -30,7 +30,7 @@ save_MICROSY
     _name.category_id             MICROSYMPOSIUM_DIC
     _name.object_id               MICROSY
 
-    _description.example.case
+    _description_example.case
```

Comment:
Each data item should have a data value associated with it; just a data name (the tag) is not enough...